pmd-apex/src/test/java/net/sourceforge/pmd/lang/apex/ast/ApexParserTest.java

```diff
@@ -191,8 +228,6 @@ private int visitPosition(Node node, int count) {
         return result;
     }
 
-    // TEST HELPER
-
     private static void assertPosition(Node node, int beginLine, int beginColumn, int endLine, int endColumn) {
```

Comment:
Just something to care about while merging into pmd 7, this method has been moved to pmd-lang-test IIRC